ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_initiator_group.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+# author alok ranjan (alok.ranjan2@hpe.com)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+author:
+  - Alok Ranjan (@ranjanal)
+description: On HPE Nimble Storage array- Create or delete initiator group and initiators
+module: hpe_nimble_initiator_group
+options:
+  access_protocol:
+    choices:
+    - iscsi
+    - fc
+    required: False
+    type: str
+    description:
+    - Initiator group access protocol.
+  app_uuid:
+    required: False
+    type: str
+    description:
+    - Application identifier of initiator group.
+  description:
+    required: False
+    type: str
+    description:
+    - Text description of initiator group.
+  fc_initiators:
+    required: False
+    type: list
+    default: None
+    description:
+    - List of FC initiators. When create/update fc_initiators, wwpn is required.
+  fc_tdz_ports:
+    required: False
+    type: list
+    description:
+    - List of target fibre Channel ports with target driven zoning configured on this initiator group.
+  host_type:
+    required: False
+    type: str
+    description:
+    - Initiator group host type. Available options are auto and hpux. The default option is auto. This attribute will be
+    - applied to all the initiators in the initiator group. Initiators with different host OSes should not be kept in the
+    - same initiator group having a non-default host type attribute.
```

Comment:
Why is this multi-line cut into multiple list entries?

Reply:
will remove ..